src/clsourcedependencies_parser.cc

```diff
@@ -1,16 +1,6 @@
-// Copyright 2020 Google Inc. All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
```

Comment:
You should keep this header.

Reply:
I work for Microsoft and just chatted with our legal team about this header. Without a Contributor License Agreement (CLA) in place for this project to affirmatively assign the copyright to Google, I can't include this line:

> Copyright 2020 Google Inc. All Rights Reserved.

Would you be okay with the first line instead reading

> Copyright 2020 Microsoft Inc. All Rights Reserved.

followed by the same Apache 2.0 License terms? If we can use this language, the Microsoft legal team is okay not having a CLA in place.

Reply:
Not for me to decide, sorry.

Reply:
@jhasse Then ping the people who can make the decision.

Reply:
I don't know (and don't care), sorry.

Reply:
You should care, you generate intellectual property for a living. Google doesn't own the copyright of any third-party contributions without copyright assignment. The copyright notice in all files that have had contributions should really be "Copyright Google and Contributors" at the very least.

Reply:
Without a CLA or revised copyright notice, I'll be forced to maintain this functionality in a fork instead of contributing it here. Any company that wants to contribute to Ninja will face this same problem, not just Microsoft. I'd _really_ like to avoid that outcome because the functionality is broadly useful and forking will fragment the community. Is there truly no one else I can contact to discuss this issue?

Reply:
Google employee here. I was a bit confused by this thread so I ran it by our OpenSource Compliance team.

Essentially, Google doesn't own this project so this is up to the maintainers, but we don't request or desire for a Google copyright notice to be added to this file.

Reply:
I added the header back with a Microsoft copyright notice.